File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_set_float_knob.rs


pub fn convert__command__set_float_knob__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command::SetFloatKnob) -> crate::victron_controller::dashboard::command::SetFloatKnob {
    crate::victron_controller::dashboard::command::SetFloatKnob {
        knob_name: from.knob_name.clone(),
        value: from.value.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_command_set-float-knob.ts
// @ts-nocheck
import {SetFloatKnob as v0_1_0_SetFloatKnob} from '../v0_1_0/Command'
import {SetFloatKnob as v0_2_0_SetFloatKnob} from './Command'

export function convert__command__set_float_knob__from__0_1_0(from: v0_1_0_SetFloatKnob): v0_2_0_SetFloatKnob {
    return new v0_2_0_SetFloatKnob (
        from.knob_name,
        from.value
    )
}